src/components/Application.pcss

```diff
@@ -67,23 +67,22 @@ code {
 
 a {
     text-decoration: none;
-    color: var(--blue-accent);
+    color: var(--main-color-text);
     transition: 0.1s color;
 
     &:hover {
         color: var(--blue-accent-80);
     }
 
     &:active {
-        color: var(--red-accent);
+        color: var(--blue-accent);
     }
 }
 
 details {
-    border: 2px solid var(--main-border-color);
```

Comment:
Changing these global styles spoiled `<details>` in the content area. Please, fix that.